draft-ietf-rtcweb-jsep.xml

```diff
@@ -1720,6 +1740,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>For MediaStreamTracks that are still present, the "a=msid",
               "a=ssrc", and "a=ssrc-group" lines MUST stay the same.</t>
 
+	      <t>For RtpTransceivers that are still present, the "a=rid" lines
+	      MUST follow the rules for subsequent offers defined in <xref
+	      target="I-D.ietf-mmusic-rid"></xref>.</t>
```

Comment:
It might be worthwhile to summarize the guidance from the other document, as done for other attributes.